src/fsharp/infos.fs

```diff
@@ -3052,6 +3063,7 @@ let GetImmediateIntrinsicPropInfosOfType (optFilter,ad) g amap m typ =
             matchingProps
             |> Seq.map(fun pi -> ProvidedProp(amap,pi,m)) 
             |> List.ofSeq
+            |> List.append (getFSharpMetadata()) // to account for any intrinsic type extensions
```

Comment:
Curried List.append is hard to read and a bit of an anti-pattern - it adds the argument at the front but people may not expect that.  I'd rather you use ``[ yield! xs; yield!ys ]`` or an explicit List.append.